src/features/pilot_management/PilotSheet/components/LevelEditDialog.vue

```diff
@@ -20,35 +20,33 @@
         <v-col cols="auto">
           <span class="overline">Current Level:</span>
           <br />
-          <v-text-field
-            v-model="pilot.Level"
-            readonly
-            type="number"
-            hide-details
-            dense
-            outlined
-            background-color="panel"
+          <span
+            style="display:inline-block"
             class="level-input"
-          />
+          >
+            {{ pilot.Level }}
+          </span>
         </v-col>
 
         <v-col cols="auto" class="mx-3">
+          <span class="overline"></span>
```

Comment:
This is literally just a placeholder to center the arrow between Current and New level.